src/mercury-inventory/mercury/inventory/server.py

```diff
@@ -34,6 +34,10 @@ def __init__(self, bind_address):
         return await self.dispatcher.dispatch(message)
 
 
+def main():
```

Comment:
```python
def main():
    logging.basicConfig(level=logging.DEBUG)
    loop = zmq.asyncio.ZMQEventLoop()

 if __name__ == '__main__':
    main()
```

Just for copy pasta if that's where you were going iwth it

Reply:
This PR isn't complete, I was going to add  argument parsing .. somewhere